tests/installation/partitioning_raid.pm

```diff
@@ -159,6 +159,11 @@ sub run {
     # create partitioning
     send_key(is_storage_ng() ? $cmd{expertpartitioner} : $cmd{createpartsetup});
 
+    if (sle_version_at_least '15') {
+        assert_screen 'expertpartitioner_list';
+        send_key 'tab';
+        send_key 'ret';
```

Comment:
We had bad experience when used send_keys without verifying that next window is there, let's wrap it in wait_screen_change at least, or we may end up in situation when we press next button before it is there.